cms/appresolver.py

```diff
@@ -223,18 +237,22 @@ def get_app_patterns():
         if title.page_id not in hooked_applications:
             hooked_applications[title.page_id] = {}
         app = apphook_pool.get_apphook(title.page.application_urls)
+        if not app:
```

Comment:
is this intentional? doing this would mean that one missing apphook interrupts the rest no?